packages/fxa-settings/src/components/Security/en-US.ftl

```diff
@@ -1,3 +1,9 @@
 ## Security section of Setting
 
 security-heading = Security
+security-password =
```

Comment:
> should we leave the ID unchanged?

I'm leaning toward changing it for consistency? Lets see what l10n team says and we can document that.

Reply:
> Lets see what l10n team says and we can document that.

I'm betting they won't say anything, but that they'd prefer leaving it as-is to prevent retranslation. Maybe it's not a big deal on their end though.

I would love to know the reason behind tying strings to IDs, as in Fluent, rather than tying them to English strings, like gettext does.